agents/gemini_data_analytics/intro_gemini_data_analytics_sdk.ipynb

```
@@ -0,0 +1,1582 @@
+        "\n",
+        "try:\n",
+        "    operation = data_agent_client.create_data_agent(request=request)\n",
+        "\n",
+        "    while not operation.done():\n",
+        "        print(\"Still in progress...\")\n",
+        "        time.sleep(2)\n",
+        "\n",
+        "    if operation.done():\n",
+        "        if operation.exception():\n",
+        "            print(\"Operation failed:\", operation.exception())\n",
+        "        else:\n",
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)